src/main/java/com/work/management/web/rest/controller/EmployeeController.java

```diff
@@ -32,4 +33,14 @@
     employeeService.save(EmployeeAssembler.convert(employeeResource));
     return new ResponseEntity<>(employeeResource, HttpStatus.CREATED);
   }
+
+  @RequestMapping(method = RequestMethod.GET, value = "/{userName}", produces = "application/json; charset=UTF-8")
+  @ApiOperation(value = "Get an employee by its user name", response = EmployeeResource.class)
+  ResponseEntity<EmployeeResource> getEmployeeByUserName(
+      @Valid @ApiParam(value = "User Name", required = true) @PathVariable("userName") String userName) {
+
+    EmployeeResource employeeResource = EmployeeAssembler
+        .convert(employeeService.getEmployeeByUserName(userName));
```

Comment:
Inline this variable(employeeResource)...cltr+shift+i...as your are not using it anywhere else...and remove the empty line at 41.